src/config.c

```diff
@@ -343,7 +343,7 @@ typedef struct {
 	git_config_iterator *current;
 	const git_config *cfg;
 	regex_t regex;
-	int has_regex;
+	int have_regex;
```

Comment:
because multivar_iter has a flag "have_regex". I thought they should be consistent. But now I realize the difference may make sense and "have" is used for multiple entries. Is that true?